src/main/java/edu/harvard/iq/dataverse/util/URLTokenUtil.java

```diff
@@ -194,6 +207,58 @@ private String getTokenValue(String value) {
         throw new IllegalArgumentException("Cannot replace reserved word: " + value);
     }
     
+    public JsonObjectBuilder createPostBody(JsonObject params, JsonArray allowedApiCalls) {

```

Comment:
This is just a refactoring to move more functionality out of the ExternalToolHandler so it can be used with the dataverse-globus app (which is similar to an external tool but is launched from the upload pane or download menus.)